hub/constants.py

```diff
@@ -37,10 +37,9 @@
 
 CHUNKS_FOLDER = "chunks"
 
-CHUNK_EXTENSION = "npz"
 ENCODED_CHUNK_NAMES_FOLDER = "chunks_index"
 # unsharded naming will help with backwards compatibility
-ENCODED_CHUNK_NAMES_FILENAME = f"unsharded.{CHUNK_EXTENSION}"
+ENCODED_CHUNK_NAMES_FILENAME = f"unsharded"
```

Comment:
if it is now sharded, should we rename this to `sharded`? or do we still only have 1 single file?

Reply:
It's not sharded in that sense, it's only sharded in memory but we end up with only a single file.